app/assets/stylesheets/application_rmc-legacy.scss

```diff
@@ -21,7 +21,6 @@
     // Center masthead image
     background-position: center;
     border: none;
```

Comment:
What about also adding `border: none` to our default styling? (and remove from rmc-legacy and dr-joyce-brothers custom themes). Looks like a border-right was only added to spotlight because of the blur: https://github.com/projectblacklight/spotlight/blob/main/app/assets/stylesheets/spotlight/_header.scss#L102-L103, which we've removed in our instance (https://github.com/cul-it/exhibits-library-cornell-edu/blob/dev/app/assets/stylesheets/exhibits/_variables_overrides.scss).

Reply:
Fixed in latest commit